Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml

```diff
@@ -0,0 +1,598 @@
+        "position": {
+          "x": 100.5,
+          "y": 785
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "12":
+    id: "12"
+    taskid: 266c8054-586a-4b62-811b-d13147f12e54
+    type: regular
+    task:
+      id: 266c8054-586a-4b62-811b-d13147f12e54
+      version: -1
+      name: Get email address info from Active Directory
+      description: Uses Active Directory to get user information for internal email
+        addresses.
+      script: '|||ad-get-user'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      attributes: {}
+      custom-field-data: {}
+      custom-field-type: {}
+      dn: {}
+      email:
+        complex:
+          root: Account
+          accessor: Email
+          transformers:
+          - operator: WhereFieldEquals
+            args:
+              equalTo:
+                value:
+                  simple: Internal
+              field:
+                value:
+                  simple: NetworkType
+              getField:
+                value:
+                  simple: Address
+          - operator: uniq
+      limit: {}
+      name: {}
+      user-account-control-out: {}
+      username: {}
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 672.5,
+          "y": 1195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "13":
+    id: "13"
+    taskid: 2d740d11-a326-4913-83f1-8df803c0a3eb
+    type: condition
+    task:
+      id: 2d740d11-a326-4913-83f1-8df803c0a3eb
+      version: -1
+      name: Are there any external email addresses?
+      description: Checks whether there are email addresses with a NetworkType attribute
+        value of "External".
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "11"
+    scriptarguments:
+      value:
+        complex:
+          root: Account
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Account.Email.NetworkType
+                iscontext: true
+              right:
+                value:
+                  simple: External
+              ignorecase: true
+          accessor: Email
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 100.5,
+          "y": 580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "14":
+    id: "14"
+    taskid: e521733d-b152-4a1f-8050-e7658405a187
+    type: condition
+    task:
+      id: e521733d-b152-4a1f-8050-e7658405a187
+      version: -1
+      name: Are there any internal email addresses?
+      description: Checks whether there are email addresses with a NetworkType attribute
```

Comment:
Checks whether there is at least one email address with a NetworkType attribute set to "Internal".